File: WoofIOS/SwiftPackages/Presentation/Favorites/Sources/PresentationFavorites/UIModel/FavoriteCellUIModel.swift
import Models

public struct FavoriteCellUIModel: Hashable, Identifiable {
    public let id: Int
    public let name: String
    public let photoUrl: String
    let domain: Dog

    init(id: Int, name: String, photoUrl: String, domain: Dog) {
        self.id = id
        self.name = name
        self.photoUrl = photoUrl
        self.domain = domain
    }
}

File: WoofAndroid/presentation/favorites/src/main/kotlin/com/perrystreet/woof/presentation/favorites/uimodel/FavoriteCellUIModel.kt
package com.perrystreet.woof.presentation.favorites.uimodel

import com.perrystreet.woof.models.dog.Dog

data class FavoriteCellUIModel(
    val id: Long,
    val name: String,
    val photoUrl: String,
    internal val domain: Dog,
)
